.azure-pipelines/template-clean-sonic-slave.yml

```diff
@@ -0,0 +1,13 @@
+steps:
+- script: |
+    containers=$(docker container ls | grep "sonic-slave" | awk '{ print $1 }')
+    if [ ! -z "$containers" ]; then
+      docker container kill $containers || true
+      sleep 5
+    fi
+    docker images | grep "^<none>" | awk '{print$3}' | xargs -i docker rmi {}
```

Comment:
Can we add a some script to remove the killed containers?
containers=$(docker container ls -a | grep "sonic-slave" | awk '{ print $1 }')
docker container rm containers

Reply:
docker container rm 3df0d0091623 dde8eb8bff64
Error response from daemon: You cannot remove a running container 3df0d0091623ecf4237d9ba3b878206af26318e882a15ee8ed9eb0d77b596ae8. Stop the container before attempting removal or force remove